cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -99,7 +74,15 @@ const (
   # You can then push your bundle image:
   $ make docker-push IMG=$BUNDLE_IMG
 `
-)
+
+// kustomization.yaml file contents for manifests. This should always be written to
+// config/bundle/kustomization.yaml since it only references files in config.
+const manifestsKustomization = `resources:
+- ../default
+- ../samples
```

Comment:
kubectl apply -f config/samples/
error: error validating "config/samples/kustomization.yaml": error validating data: [apiVersion not set, kind not set]; if you choose to ignore these errors, turn validation off with --validate=false